internal/repository/downstreams/attendeeservice/mock.go

```diff
@@ -0,0 +1,85 @@
+package attendeeservice
+
+import (
+	"context"
+	"errors"
+	"github.com/eurofurence/reg-room-service/internal/repository/downstreams"
+	"github.com/eurofurence/reg-room-service/internal/web/common"
+)
+
+type Mock interface {
+	AttendeeService
+
+	Reset()
+	Unavailable()
+	SetupRegistered(subject string, badgeNo int64, status Status)
+}
```

Comment:
These kind of things are actually something that I wouldn't like to maintain. There are really nice mocking frameworks that always will generate mock definitions whenever an interface changes. 

Can we have a chat and talk about using one of the existing ones?

Reply:
Moved to #70 -resolving here